src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2320,35 +2320,49 @@ void func_80833A20(Player* this, s32 newMeleeWeaponState) {
 
 s32 func_80833B2C(Player* this) {
     if (this->stateFlags1 & (PLAYER_STATE1_16 | PLAYER_STATE1_PARALLEL | PLAYER_STATE1_LOCK_ON_FORCED_TO_RELEASE)) {
-        return 1;
+        return true;
     } else {
-        return 0;
+        return false;
     }
 }
 
-s32 func_80833B54(Player* this) {
+/**
+ * Checks the current state of `focusActor` and if it is a hostile actor (if applicable).
+ * If so, sets `PLAYER_STATE1_HOSTILE_LOCK_ON` which will control Player's "battle" response to
+ * hostile actors. This includes affecting how movement is handled, and enabling a "fighting" set
+ * of animations.
+ *
+ * Note that `Player_CheckHostileLockOn` also exists to check if there is currently a hostile lock-on actor.
+ * This function differs in that it first updates the flag if appropriate, then returns the same information.
+ *
+ * @return  true if there is curerntly a hostile lock-on actor, false otherwise
```

Comment:
I'm a bit late but there's a typo `curerntly`

Reply:
will get in the next related PR, thanks